doc/nrf/releases_and_maturity/abi_compatibility.rst

```diff
@@ -54,6 +57,287 @@ Additionally, maintaining ABI compatibility allows the nRF54H20 SoC binary compo
     The nRF54H20 SoC binaries only support specific versions of the |NCS| and do not support rollbacks to previous versions.
     Upgrading the nRF54H20 SoC binaries on your development kit might break the DK's compatibility with applications developed for earlier versions of the |NCS|.
 
+nRF54H20 SoC binaries v22.2.0+14 changelog
+******************************************
+
+The following sections provide detailed lists of changes by component.
+
+IronSide Secure Element (IronSide SE) v22.2.0+14
+================================================
+
+Added
+-----
+
+* Support for disabling and enabling USB D+ pull-up control to ensure VBUS-detected IRQs are received.
+
+Changed
+-------
+
+* Enabled IPC unbound feature.
+* Improved power consumption.
+* Improved stability.
+
+Removed
+-------
+
+* Temperature subscription
+
+IronSide Secure Element (IronSide SE) v22.1.0+13
+================================================
+
+Added
+-----
+
+* NCSDK-33486: Support for configuring TDD (CoreSight++) from local domains.
```

Comment:
Still a few items starting with the Jira issue. Move all to the end in brackets as you have done in most items.